scripts/hf_sim.py

```diff
@@ -163,63 +169,132 @@ def random_seed():
         default=["1", "-1", "-1"],
     )
 
-    try:
-        args = parser.parse_args()
-    except SystemExit as e:
-        print(e, flush=True)
-        # invalid arguments or -h
-        comm.Abort()
+    args = parser.parse_args(cmd)
```

Comment:
Clever, leaving it as None results in default behaviour. Maybe add that to the docstring though?